<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>499</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>499</width>
    <height>441</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Save To</string>
     </property>
     <widget class="QLineEdit" name="le_saveLocation">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>20</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_browse">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>60</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_saveLocation">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>23</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>At here</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_askMe">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ask me everytime</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>451</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Interface</string>
     </property>
     <widget class="QCheckBox" name="cb_toTray_onClose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to tray on closing the window</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_toTray_onStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to tray on start</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_always_trayIcon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Show tray icon</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="cb_imgFormat">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>110</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jpeg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gif</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>113</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Format</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Default Save Action</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_defaultAction">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>147</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Always ask</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Edit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Save to file</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Save to Clipboard</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Style</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Font</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose font for Text Item:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_textItemFont">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>451</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QPushButton" name="pb_itemColor">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_interfaceColor">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Default Item Color :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Default Background Color :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rb_saveLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>pb_browse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_saveLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_saveLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
